Controller/Payment/Checkout/Token.php

```diff
@@ -122,4 +141,28 @@ public function execute()
             ->setHttpResponseCode(\Magento\Framework\Webapi\Response::HTTP_OK)
             ->setData($resBody);
     }
+
+    /**
+     * CreateBuyer
+     *
+     * @param $transactionId
+     */
+    public function createBuyer($transactionId)
```

Comment:
Let's move all buyer related functionality to more appropriate class (like `BalanceBuyer` class)